python/neuroglancer/pipeline/storage.py

```diff
@@ -132,9 +115,16 @@ def get_files(self, file_paths):
 
         results = []
 
-        def store_result(path, result, error):
-            content, decompress = result
+        def get_file_thunk(path, interface):
+            result = error = None 
 
+            try:
+                result = interface.get_file(path)
+            except Exception as err:
+                error = err
+                print(err)
```

Comment:
In this case, it ensures that all exceptions can be handled per file since they are placed in an array describing the error.